robottelo.properties.sample

```diff
@@ -25,6 +25,13 @@ ssh_key=
 # Admin password when accessing API and UI
 # admin_password=changeme
 
+# section for ssh client settings
+[ssh_client]
```

Comment:
And because there's no validation you've missed an issue here :) It should be commented by default - all the custom sections are either fully commented out or fully uncommented